File: backend/SmakApi/Models/DTOs/IngredientItemDto.cs
using SmakApi.Models.Entities;

namespace SmakApi.Models.DTOs;

public class IngredientItemDto
{
    public string Name { get; set; } = null!;
    public float? Quantity { get; set; }
    public UnitOfMeasure? Unit { get; set; }
}

File: mobile/app/src/main/java/com/itstep/smakmobile/dto/IngredientItemDto.java
package com.itstep.smakmobile.dto;

public class IngredientItemDto {
    public String name;
    public Float quantity;
    public String unit;

    public String getName() {
        return name;
    }

    public Float getQuantity() {
        return quantity;
    }

    public String getUnit() {
        return unit;
    }

    public String getAmountWithUnit() {
        return String.format("%.0f %s", quantity, unit);
    }
}
